BeagleBone_black/src/navigation/navigation.hpp

```diff
@@ -72,6 +73,10 @@ constexpr std::array<NavigationType, 42> kStripeLocations = {0.0,
      640.08,  670.56,  701.04,  731.52,  762.0,  792.48,  822.96,  853.44,  883.92,  914.4,
      944.88,  975.36, 1005.84, 1036.32, 1066.8, 1097.28, 1127.76, 1158.24, 1188.72, 1219.2,
     1249.68};
+constexpr std::array<float, 16> coefficients = {80.7074,
```

Comment:
This should be called `kCoefficients`. I'd also move it to the `getBrakingDistance` function since it doesn't make any sense outside of it - it's just a bunch of weird numbers.

Reply:
wouldn't that initialise an array every time the function is called though?